lib/config/options/index.ts

```diff
@@ -2408,6 +2408,17 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'locale',
+    description: `User-facing text language`,
+    type: 'string',
+    default: 'en',
+  },
+  {
+    name: 'translationsFilePath',
+    description: `PO (Portable Object) file's path`,
+    type: 'string',
+  },
```

Comment:
Is it intended that this should be configurable per-repo by individual users, or that it is configured "per bot" and controlled by an administrator? It looks like it should be the latter, which means the above config options should be configured with globalOnly=true

Reply:
OK, I will modify those options's scope.